apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -77,323 +77,266 @@ export function InstructorQuestionSettings({
+                            {
+                              id_label: 'QID',
+                              buttonID: 'changeQidButton',
+                              id_old: resLocals.question.qid,
+                              ids: qids,
+                              ...resLocals,
+                            },
+                          )}"
+                          data-trigger="click"
+                        >
+                          <i class="fa fa-i-cursor"></i>
+                          <span>Change QID</span>
+                        </button>
+                      `
+                    : ''
+                }
+                ${
+                  questionGHLink
+                    ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
+                    : ''
+                }
+                <input
+                  type="text"
+                  class="form-control"
+                  id="qid"
+                  name="qid"
+                  value="${resLocals.question.qid}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is a unique identifier for the question. (e.g., "addNumbers")
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="type">Type</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="type"
+                  name="type"
+                  value="${resLocals.question.type}"
+                  disabled
+                />
+                <small class="form-text text-muted"> This is the type of question. </small>
+              </div>
+              <div class="form-group">
+                <div>Topic</div>
+                <div name="topic">
+                  ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                    topic: resLocals.topic,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>Tags</div>
+                <div name="tags">
+                  ${renderEjs(__filename, "<%- include('../partials/tags') %>", {
+                    tags: resLocals.tags,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>
+                  Issues
+                  ${renderEjs(__filename, "<%- include('../partials/issueBadge') %>", {
+                    count: resLocals.open_issue_count,
+                    issueQid: resLocals.question.qid,
+                    suppressLink: resLocals.suppressLink,
+                    urlPrefix: resLocals.urlPrefix,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>Assessments</div>
```

Comment:
This bit looks funky when there aren't any issues/assessments:

<img width="239" alt="Screenshot 2024-04-17 at 12 11 47" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/648fa954-dfbf-4959-b877-128d56da036f">

A few ideas:

- We might consider actually adding an "issues" tab to the instructor question view? That way we could get this out of this page entirely (it's a bit weird to have a link to a list of issues on a "settings" page IMO.
- We could show the list of assessments with a Bootstrap list group, and maybe show extra information, e.g. the course instance. We could then have a nice empty state when there aren't any, e.g.:

  <img width="767" alt="Screenshot 2024-04-17 at 12 17 26" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/30e5f1e5-b1e9-44a8-909b-fdd483ca7d32">

Reply:
Added an "issues" tab in [f9945bf](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/f9945bf0e4c4e0d20d0a34f07c433d9e135dc0c7): <img width="895" alt="Screenshot 2024-04-18 at 5 00 08 PM" src="https://github.com/PrairieLearn/PrairieLearn/assets/114321556/39df91f3-d617-44b1-b927-5722168fa447">
Currently, this new tab just links to the `instructorIssues` page with a `query param` of `qid%3A{question.qid}`. This will take the user to the existing issues page and filter by the current qid. I can build a specific `questionIssues` page in a future PR if this is a bit funky, but that seemed like a large endeavor to include in this PR and this seems to provide the functionality we would want here. 



The assessments section now looks like this: 
<img width="1290" alt="Screenshot 2024-04-18 at 5 05 03 PM" src="https://github.com/PrairieLearn/PrairieLearn/assets/114321556/f98e1b5f-0438-48a9-9a91-c9493a21e689">
Assessments are grouped by course instance and displayed in a card for readability. I'm not certain on the columns we want included though. Currently I have `label/badge`, `type`, and `title`. Let me know if you would prefer any other columns here or to remove the `type` column if that is redundant (or if I should order these differently?). 

Also, questions that are not included in any assessments now show the following: 
![Screenshot 2024-04-18 at 5 11 54 PM](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/3e48dcaa-ab8c-4367-b992-5e85f8628807)